packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -30,37 +26,20 @@ const stubCreateEthers = (ctrl: TokensController, res: boolean) => {
 describe('TokensController', () => {
   let tokensController: TokensController;
   let preferences: PreferencesController;
-  let network: NetworkController;
-  let messenger: NetworkControllerMessenger;
 
   let instEthProvStub: sinon.SinonStub;
+  let onNetworkStateChangeCallback: any;
+
+  const rinkeby = { chainId: '4', type: 'rinkeby' } as ProviderConfig;
```

Comment:
Instead of typecasting, is it possible to provide a full provider config object? Otherwise these tests are lying (or if they aren't, we can't tell).